wiki/osu!_originals/fr.md

```diff
@@ -0,0 +1,476 @@
+# osu! originals
+
+Les **osu! originals** sont des musiques spécifiquement faites pour (ou ont fait leur première apparition sur) osu! ou d'autres événements en rapport avec osu! tels que les tournois, les concours, et les annonces de [Featured Artists](/wiki/Featured_Artists). Bien que les osu! originals ne soient pas toutes sous licence officielle ou dans la [liste des Featured Artists](https://osu.ppy.sh/beatmaps/artists), les musiques qui le sont sont généralement distinguées par le label "ORIGINAL" sur les pages de leurs artistes respectifs.
+
+Toutes les osu! originals qui ont été autorisées à l'utilisation par le jeu peuvent être utilisées sur osu! à des conditions non exclusives conformément à la [politique de licence musicale d'osu!](/wiki/Legal/Music_licensing). À l'inverse, les osu! originals qui n'ont pas été autorisées à l'utilisation peuvent nécessiter une autorisation d'utilisation. Voir les [Directives d'utilisation du contenu](/wiki/Rules/Content_Usage_Guidelines) pour plus d'informations.
```

Comment:
> les osu! originals qui ont été autorisées à l'utilisation par le jeu peuvent être utilisées sur osu!

> les osu! originals qui n'ont pas été autorisées à l'utilisation peuvent nécessiter une autorisation d'utilisation

lolwat

i mean, yes, this is technically correct, but can't it be expressed in a better way? perhaps using the word "licence"?